test/TokenTestUtils.js

```diff
@@ -216,6 +222,23 @@ function buildExpectedState(token, customVars) {
     return expectedState;
 }
 
+// Replaces all BN objects with 32 character hex strings
+function mapBNToHex(state) {
```

Comment:
Perhaps here it might be easier to use the web3 tohex function that can take BN to hex (https://web3js.readthedocs.io/en/1.0/web3-utils.html#tohex)?

Reply:
Will do this for all BN to hex conversions.